src/hunspell/hunspell.cxx

```diff
@@ -624,7 +624,8 @@ bool HunspellImpl::spell(const std::string& word, int* info, std::string* root)
         return false;
       return true;
     }
-    return true;
+
+    goto success;
```

Comment:
I think wrapping HunspellImpl::suggest() will also fix the XML API's lack of OCONV. (https://github.com/hunspell/hunspell/issues/88#issuecomment-119567510)